scripts/histmakers/mw_lowPU.py

```diff
@@ -23,14 +22,12 @@
 
 corr_helpers = theory_corrections.load_corr_helpers(common.wprocs_lowpu, args.theory_corr)
 
-filt = lambda x,filts=args.filterProcs: any([f in x.name for f in filts]) 
-excludeGroup = args.excludeProcGroups if args.excludeProcGroups else None
 datasets = wremnants.datasetsLowPU.getDatasets(maxFiles=args.maxFiles,
-                                               filt=filt if args.filterProcs else None,
-                                               excludeGroup=excludeGroup,
-                                               flavor=flavor)
+                                              filt=args.filterProcs,
+                                              excl=args.excludeProcs, 
+                                              flavor=flavor)
 
-logger = common.setup_logger(__file__, args.verbose, args.color_logger)
+logger = logging.setup_logger(__file__, args.verbose, args.color_logger)

```

Comment:
Should we maybe use a different name to avoid the ambiguity with the main python logging module?

Reply:
Could either be a different name for the file (logger, wremlogging) or we always import it with another name (as logger, as log etc)

Reply:
I was thinking of our logging as a replacement for the python logging module. In that way, we don't end up that someone using accidentally both in one script.

Reply:
That's true, ok